frontend/src/metabase/lib/redux/hooks.ts

```diff
@@ -0,0 +1,31 @@
+import type { AnyAction } from "redux";
+import {
+  useDispatch as useDispatchOriginal,
+  useSelector as useSelectorOriginal,
+} from "react-redux";
+import type { TypedUseSelectorHook } from "react-redux";
+
+import type { State } from "metabase-types/store";
+
+// Note (EmmadUsmani): This interface is simply to keep typescript from erroring when
```

Comment:
will you keep this note? if so, I'd vote for removing mentioning you, it's pretty easy to find the author from git history

Reply:
Yup I will keep the comment. Regarding the author, I've followed this convention at my previous company and found that it has a lot of value. Oftentimes there will be large refactors for things like enabling lint rules or changing file structure, so it helps to explicitly name the author to prevent it from getting buried too deep in the git history.

Reply:
Good point, that's probably better. It's a bit awkward in situations like this where there is no issue, but you can just push a draft PR first then copy the PR number. I've just updated the comment with this.